dev/io.openliberty.microprofile.metrics.internal.3.0.monitor/resources/io/openliberty/microprofile/metrics/internal/monitor/resources/MonitorMetrics.nlsprops

```diff
@@ -24,4 +24,8 @@ FEATURE_REGISTERED.useraction=No user action is required.
 
 FEATURE_UNREGISTERED=CWPMI2004I: All monitoring metrics were unregistered from mpMetrics.
 FEATURE_UNREGISTERED.explanation=The Performance Monitoring feature, the MicroProfile Metrics feature, or both features were removed from the server. All monitoring metrics were unregistered from MicroProfile Metrics.
-FEATURE_UNREGISTERED.useraction=No user action is required.
\ No newline at end of file
+FEATURE_UNREGISTERED.useraction=No user action is required.
+
+METRICS_UNHANDLED_JAXRS_EXCEPTION=CWPMI2005W: The MicroProfile Metrics exception mapper detected and is handling an unhandled exception from the JAX-RS application.
+METRICS_UNHANDLED_JAXRS_EXCEPTION.explanation=The JAX-RS-based application experienced an exception that is not handled within the application. The MicroProfile code handles it by logging the exception stack trace and returning a 500 Internal Server Error response.
+METRICS_UNHANDLED_JAXRS_EXCEPTION.useraction=This behavior might be acceptable. To handle the exception differently, the application developer can add an ExceptionMapper interface that handles the exception and maps it to a different response.
```

Comment:
Suggested update" Remove "the application developer can"

Reply:
resolved